src/components/new-request/search.vue

```diff
@@ -274,40 +242,7 @@
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
         <BusinessLookupFetch ref="MyBusinessLookup"/>
-        <v-dialog  v-if="isSocietyDisabled" v-model="showSocietiesInfo"
-          :max-width="maxWidth"
-          persistent>
-          <v-card>
-              <v-icon
-                class="float-right"
-                md
-                color="primary"
-                @click="closeSocietyDialog()"
-              >
-              mdi-close
-              </v-icon>
-              <template>
-                <v-card-text class="mt-n12">
-                  <v-container fluid>
-                    <v-row
-                      no-gutters
-                      class="text-center"
-                    >
-                      <v-col cols="12">
-                        To request a restoration for a Society
-                      </v-col>
-                      <v-col cols="12">
-                        please use the Societies Online website
-                      </v-col>
-                      <v-col cols="12">
-                        <a href="https://www.bcregistry.ca/societies/">https://www.bcregistry.ca/societies/</a>
-                      </v-col>
-                    </v-row>
-                  </v-container>
-                </v-card-text>
-              </template>
-          </v-card>
-        </v-dialog>
+        <SocietiesInfo type='restoration' />
```

Comment:
Please check with UI/UX team (eg, Janis or Yui) on whether they want to see "To request a restoration for a Society".

Reply:
What did they say about the messages?

Reply:
Checked with Yui Sotozaki. Made the changes accordingly.

Should be "To restore a Society..."

Reply:
Thanks for the changes.

PS Instead of specifying a prop as dynamic, which required you to pass the string as an expression, you could do this.

Before:
![image](https://github.com/bcgov/namerequest/assets/10002889/9385831b-aac8-43ec-9215-ff0bf41c2601)

After:
```
  type="request a name for"
```

This works because the string is static.

There's a similar thing when passing boolean props. You can do this (note: no quotes):
```
  myProp=true
```
or this (dynamic prop so what is in quotes is evaluated):
```
  :myProp="true"
```

Reply:
I'm resolving this conversation because your existing code is OK.